### PR TITLE
Update avatar appearance

### LINKED ISSUE #1330: Updates to avatar appearance
some minor UI tweaks to the avatar views:
1. Lighter contributor font, let's do a font-weight of 300
2. "Release Maintainers & Contributors" is too wordy. Open to suggestions...
3. Remove the green from the new contributors, make it like a regular contributor, and change it to  "Contributor - New" as the text.
4. Have the maintainers first, but in the same row as the contributors, but the author should be on a seperate row
5. Even if there is no release update the Author and maintainer should still be displayed if available. let's make the font-weight 400

### PR DESCRIPTION
**Pull request by brianjp93**
- fixes #1330

### CODE REVIEW

**Comment by brianjp93** [comment]
image

**Comment by brianjp93** [comment]
Author will be added in separate PR

**Comment by rbbeeston** [comment]
Looks good to me. Vinnie and I discussed the titles and would like to replace "Release Maintainers & Contributors" with "This Release" and replace "Previous Contributors" with "All Time"

**Review by rbbeeston — APPROVED** [review]

**Comment by vinniefalco** [comment]
NOICE
